c2cgeoportal/lib/cacheversion.py

```diff
@@ -61,7 +61,8 @@ def __call__(self, request):
 
         response = self.handler(request)
 
-        response.headers["Access-Control-Allow-Origin"] = "*"
-        response.headers["Access-Control-Allow-Headers"] = "X-Requested-With, Content-Type"
+        if path[1] in CACHE_PATH:
```

Comment:
Yes, but I don't understand why you do that only if path[1] in CACHE_PATH?

And, bonus question, why not in the previous if with exactly the same condition?